<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LinhLe" failures="3" tests="4" name="Pet Test" time="9.819" errors="0" timestamp="17 thg 9 2023 07:11:11 GMT">
  <testcase classname="petStore.api.test.DataDrivenTests" name="testPostUser" time="0.0">
    <skipped/>
  </testcase> <!-- testPostUser -->
  <testcase classname="petStore.api.test.DataDrivenTests" name="testDeleteUserByName" time="3.837">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at petStore.api.test.DataDrivenTests.testDeleteUserByName(DataDrivenTests.java:30)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="petStore.api.test.DataDrivenTests" name="testDeleteUserByName" time="1.778">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at petStore.api.test.DataDrivenTests.testDeleteUserByName(DataDrivenTests.java:30)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="petStore.api.test.DataDrivenTests" name="testDeleteUserByName" time="1.796">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at petStore.api.test.DataDrivenTests.testDeleteUserByName(DataDrivenTests.java:30)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- Pet Test -->
